lib/smart_proxy_ansible/plugin.rb

```diff
@@ -9,6 +9,17 @@ class Plugin < Proxy::Plugin
 
       settings_file 'ansible.yml'
       plugin :ansible, Proxy::Ansible::VERSION
+
+      after_activation do
+        begin
+          require 'smart_proxy_dynflow_core'
```

Comment:
Do we require smart_proxy_dynflow_core, because we do not want to initialize settings if Dynflow is standalone?

Reply:
yes